contracts/DPoS.sol

```diff
@@ -464,7 +489,7 @@ contract DPoS is IDPoS, Ownable, Pausable, WhitelistedRole, Govern {
     function withdrawFromUnbondedCandidate(address _candidateAddr, uint256 _amount)
         external
         onlyNonZeroAddr(_candidateAddr)
-        minAmount(_amount, 1 ether)
+        minAmount(_amount, 1 * DECIMALS_MULTIPLIER)
```

Comment:
Why prefer DECIMALS_MULTIPLIER  here? Make the code easier to understand?

Reply:
yes, avoid confusion, as it is CELR not ether,